hack/test-go.sh

```diff
@@ -92,6 +94,12 @@ shift $((OPTIND - 1))
 # Use eval to preserve embedded quoted strings.
 eval "goflags=(${KUBE_GOFLAGS:-})"
 
+if [[ -n "${1-}" ]]; then
+  test_dirs=$@
+else
+  test_dirs=$(kube::test::find_dirs)
```

Comment:
Instead of using a test_dirs variable you should just keep using `$@` but then set it to the output of find_dirs here.

```shell
  set -- $(kube::test::find_dirs)
```

The location moving up is probably wrong though, because the next step checks for arguments starting with `-*` and splits them in goflags and testcases so I'm wondering why you'd want a test_dirs that potentially includes `$@` including the `-*` arguments.